src/components/IconButton/docs/index.story.tsx

```diff
@@ -38,8 +38,12 @@ const iconExamples = [
   },
 ];
 
-const code = config =>
-  baseCode({ components: allComponents, compact: true, ...config });
+const example = (config, extraContext = {}) =>
+  baseExample({
+    components: { ...allComponents, ...extraContext },
+    compact: true,
+    ...config,
+  });
 
 export default {
   category: StoryCategory.COMPONENTS,
```

Comment:
If you are already making some changes here maybe you could add a description for that component :)

Reply:
💪